feat(redshift-alpha): implement IGrantable with a default service IAM Role (#28018)
aws/aws-cdk: packages/@aws-cdk/aws-redshift-alpha/README.md

### Granting Privileges

> You can grant a *cluster* IAM permissions to perform actions on other AWS services.
> For example, you may want to allow a cluster to read from a S3 bucket for Redshift Spectrum.
> 
> ```ts fixture=cluster
> const bucket = new s3.Bucket(stack, 'KmsBucket',{
>   encryptionKey: new kms.Key(stack,'Key')
> })
> bucket.grantRead(cluster)

Reviewer: ```suggestion
The `Cluster` construct implements `IGrantable`.

For example, you may want to allow a cluster to read from a S3 bucket for Redshift Spectrum:

```ts fixture=cluster
const key = new kms.Key(this, 'Key');
const bucket = new s3.Bucket(this, 'Bucket', {
  encryptionKey: key,
});

bucket.grantRead(cluster);
```

```

> 
> You can give a *user* privileges to perform certain actions on a table by using the

Reviewer: ```suggestion
You can give a user privileges to perform certain actions on a table by using the
```
